src/Stripe.net/Services/Accounts/AccountCreateOptions.cs

```diff
@@ -7,123 +7,50 @@ namespace Stripe
-        /// </summary>
         [JsonProperty("capabilities")]
         public AccountCapabilitiesOptions Capabilities { get; set; }
 
-        /// <summary>
-        /// Information about the company or business. This field is null
-        /// unless <see cref="BusinessType"/> is set to <c>company</c>.
-        /// </summary>
         [JsonProperty("company")]
         public AccountCompanyOptions Company { get; set; }
 
-        /// <summary>
-        /// The country in which the account holder resides, or in which the
-        /// business is legally established. This should be an ISO 3166-1
-        /// alpha-2 country code. For example, if you are in the United States
-        /// and the business for which you're creating an account is legally
-        /// represented in Canada, you would use CA as the country for the
-        /// account being created.
-        /// </summary>
         [JsonProperty("country")]
         public string Country { get; set; }
 
-        /// <summary>
-        /// Three-letter ISO currency code representing the default currency
-        /// for the account. This must be a currency that Stripe supports in
-        /// the account's country.
-        /// </summary>
         [JsonProperty("default_currency")]
         public string DefaultCurrency { get; set; }
 
-        /// <summary>
-        /// The email address of the account holder. For Custom accounts, this
-        /// is only to make the account easier to identify to you: Stripe will
-        /// never directly email your users.
-        /// </summary>
         [JsonProperty("email")]
         public string Email { get; set; }
 
-        /// <summary>
-        /// <para>
-        /// A card or bank account to attach to the account. You can provide either a token, like
-        /// the ones returned by <a href="https://stripe.com/docs/stripe.js">Stripe.js</a>, or a
-        /// <see cref="AccountBankAccountOptions"/> or <see cref="AccountCardOptions"/> instance.
-        /// </para>
-        /// <para>
-        /// By default, providing an external account sets it as the new default external account
-        /// for its currency, and deletes the old default if one exists. To add additional external
-        /// accounts without replacing the existing default for the currency, use the bank account
-        /// or card creation API.
-        /// </para>
-        /// </summary>
         [JsonProperty("external_account")]
         [JsonConverter(typeof(AnyOfConverter))]
         public AnyOf<string, AccountBankAccountOptions, AccountCardOptions> ExternalAccount { get; set; }
 
-        /// <summary>
-        /// Information about the person represented by the account. This field
-        /// is null unless <see cref="BusinessType"/> is set to <c>individual</c>.
-        /// </summary>
         [JsonProperty("individual")]
-        public PersonCreateOptions Individual { get; set; }
+        public AccountIndividualOptions Individual { get; set; }
```

Comment:
Did you forget to rebase? This was in the previous PR we already merged https://github.com/stripe/stripe-dotnet/pull/2159/files#diff-b1db3acda2c6906eb4ad36e4d5e8d7acR39